tomviz/SaveWebReaction.h

```diff
@@ -17,6 +17,7 @@
 #define tomvizSaveWebReaction_h
 
 #include <pqReaction.h>
+#include "PythonUtilities.h"
```

Comment:
This is the section I don't like @cjh1 but I don't know how to write it in a better way.